### DISCUSSION TITLE
People-based Attribution

### DISCUSSION THREAD

**Discussion by Jengro777** [plausible/analytics#3232] [Anything Else]
Can Plausible Analytics support user attribution analysis similar to what AppsFlyer provides?
refer to：https://www.appsflyer.com/

**Comment by metmarkosaric** [comment]
hello @Avey777! not familiar with AppsFlyer but you can filter your dashboard by any custom event and then see referral sources that drove the traffic that converted on that event in the Top Sources report. here's the example from our live demo where we track trial signups https://plausible.io/plausible.io?goal=Signup
